packml_msgs/msg/State.msg

```diff
@@ -0,0 +1,31 @@
+# PackML State, based on the PackML standard, summarized here:
+# http://www.plcopen.org/pages/promotion/publications/downloads/mapping_omac_statediagram.pdf
+
+# Enumeration values are based upon tag guidelines, here:
+# http://sesam-world.com/_pdf/make2pack/mode/2010-11-29/Materiale/PackML_Tag_Naming_Guidelines_V2.0.pdf
+# Due to changes in the standard some states weren't enumerated.  I've made my best guess
+# below
+
+# Values defined in tag naming guidelines
+int8 UNDEFINED = 0
+int8 OFF = 1            # Deleted in V3.0 of packml
+int8 STOPPED = 2
+int8 STARTING = 3
+int8 IDLE = 4
+int8 SUSPENDED = 5
+int8 EXECUTE = 6
```

Comment:
This may be defined by the standard, but it seems rather inconsistent (all other state names are either present continuous or past simple).